mammoth/inputters/dataset.py

```diff
@@ -15,18 +16,21 @@
 from mammoth.inputters.vocab import Vocab
 
 
+TensorWithMask = collections.namedtuple('TensorWithMask', 'tensor mask')
```

Comment:
please make the fields a list. i can already here the unfortunate temp dev / student / early phd who would lose a day trying to figure this out